cypress/plugins/index.js

```diff
@@ -47,5 +47,18 @@ module.exports = (on, config) => {
     }
   });
 
+  on('task', {
+    log(message) {
+      console.log(message)
```

Comment:
Suggest we're consistent in either having no semi-colon or always having a semi-colon.

Reply:
Done